flycheck.el

```diff
@@ -232,6 +232,7 @@ attention to case differences."
     rust-cargo
     rust
     sass
+    sass-lint
```

Comment:
Regarding priority, since `sass-lint` can be used for both `sass` and `scss` modes but it is not as developed as `scss-lint` yet, I think the priority should be:

1. scss-lint
2. sass-lint
3. sass
4. scss

If there are no objections I can make the change later today.